package.json

```diff
@@ -15,6 +15,13 @@
     "bower.json"
```

Comment:
We need to add the new type definition file to this list

Reply:
```suggestion
    "bson.ts34.d.ts"
```